conans/client/configure_environment.py

```diff
@@ -53,10 +53,44 @@ def __init__(self, *args):
         except:
             self.libcxx = None
 
+    def _gcc_arch_flags(self):
+        if self.arch == "x86_64":
+            return "-m64"
```

Comment:
Absolutely, you are quite right
For me it makes sense, but I wanted to make sure that people that consider this a feature won't get too annoyed. So far, it seems the feedback is consistent :)